libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/stats/stats_top-authors-day.json

```diff
@@ -0,0 +1,99 @@
+{
+    "request": {
+        "method": "GET",
+        "urlPath": "/rest/v1.1/sites/106707880/stats/top-authors/",
+        "queryParameters": {
+            "period": {
+                "equalTo": "month"
```

Comment:
This should be `"equalTo": "day"` instead of `"equalTo": "month"`, right? I'm getting a mocks failure because `/rest/v1.1/sites/106707880/stats/top-authors/?period=day&max=7&date=2020-01-21&locale=en_US` doesn't have a match.

Reply:
Yep! Fixed